Content.Shared/Foldable/FoldableSystem.cs

```diff
@@ -137,6 +138,9 @@ public bool TrySetFolded(EntityUid uid, FoldableComponent comp, bool state)
         if (!CanToggleFold(uid, comp))
             return false;
 
+        if (_pointLight.TryGetLight(uid, out var light)) // DeltaV - Holographic Rollerbeds emit light.
+            _pointLight.SetEnabled(uid, !state, light);
+
```

Comment:
Do you even need this if the folded entity has attribute?
```yml
  - type: PointLight
    enabled: false
```

Reply:
Yes? To actually toggle the light?

Reply:
K, just making sure we need it. Seems like a lot to add a whole check for a single item but I get that its a bit of a special case.